package.json

```diff
@@ -18,6 +18,9 @@
     "node": ">= 8.5",
     "npm": ">= 5"
   },
+  "dependencies": {
+    "@openui5/sap.ui.core": "1.66.1"
```

Comment:
Did you use a fixed version on purpose?
I would recommend to use a range with the minimum version required (e.g. `^1.66.1` or even a lower version, as I think you don't use any of the newer APIs).
But as discussed, this is something that still needs to clarified...

Reply:
You're right. We'll make a downport testing and enter this as the version number